orchagent/main.cpp

```diff
@@ -205,6 +209,15 @@ int main(int argc, char **argv)
     }
 
     try {
+        if (reinit)
+        {
+            SWSS_LOG_NOTICE("Start orchagent reinitialization");
+            orchDaemon->reinit();
+            /* After reinitialization, a second initialize_switch function is called
+             * to notify the ASIC that reinitialization is done. */
+            sai_switch_api->initialize_switch(0, "", "", &switch_notifications);
```

Comment:
take a look at sairedis implementation on initialize you need to pass pecific string to tell that